core/io/file_access.cpp

```diff
@@ -810,6 +810,9 @@ void FileAccess::_bind_methods() {
 	ClassDB::bind_static_method("FileAccess", D_METHOD("open_compressed", "path", "mode_flags", "compression_mode"), &FileAccess::open_compressed, DEFVAL(0));
 	ClassDB::bind_static_method("FileAccess", D_METHOD("get_open_error"), &FileAccess::get_open_error);
 
+	ClassDB::bind_static_method("FileAccess", D_METHOD("get_file_as_array", "path"), &FileAccess::_get_file_as_array);
```

Comment:
I think `get_file_as_bytes()` would be a better name.

Reply:
Sounds good to me.

Reply:
Renamed (it's not a new API, but was not exposed, so should not be a breaking change).